docs/documentation/concepts/things.md

```diff
@@ -7,12 +7,17 @@ layout: documentation
 Things are the entities that can physically be added to a system and which can potentially provide many functionalities in one. It is important to note that things do not have to be devices, but they can also represent a web service or any other manageable source of information and functionality.
 From a user perspective, they are relevant for the setup and configuration process, but not for the operation.
 
-Things can have configuration properties, which can be optional or mandatory. Such properties can be basic information like an IP address, an access token for a web service or a device specific configuration that alters its behavior.
+Things can have configuration properties, which can be optional or mandatory. Such properties can be basic information like an IP address, an access token for a web service or a device specific configuration that alters its behaviour.
 
-Things provide "channels", which represent the different functions they provide. Channels are linked to items, where such links are the glue between the virtual and the physical layer. Once such a link is established, a thing reacts on events sent for an item that is linked to one of its channels. Likewise, it actively sends out events for items linked to its channels.
+### Channels
+Things provide "channels", which represent the different functions the thing provides. Where the thing is the physical entity or source of information, the channel is a concrete function from this thing. A physical light bulb might have a dimmer channel and a colour channel, both providing functionality of the one light bulb thing to the system. For sources of information the thing might be the local weather with information from a web service with different channels like temperature, pressure and humidity.
```

Comment:
Could you put every sentence in a new line? This makes diffing and reviewing much easier - thanks!